Content.Shared/Body/Components/SharedBodyComponent.cs

```diff
@@ -197,7 +197,7 @@ protected virtual void OnRemovePart(BodyPartSlot slot, SharedBodyPartComponent p
             {
                 if (part.IsVital && SlotParts.Count(x => x.Value.PartType == part.PartType) == 0)
                 {
-                    damageable.ChangeDamage(DamageType.Bloodloss, 300, true); // TODO BODY KILL
+                    damageable.ChangeDamage(damageable.GetDamageType("Bloodloss"), 300, true); // TODO BODY KILL
```

Comment:
This is a tragic side effect of body component and mob state component meeting to create the ultimate display in code that keeps me up at night, but the damage type to deal can at least be moved to a data field to make it less eye grating, along with the amount as well.

Reply:
when smug bodysystem sus